agent.yml.sample.rackspace

```diff
@@ -0,0 +1,11 @@
+control-service:
+    hostname: 'localhost'
+    port: 4524
+dataset:
+    backend: "openstack"
+    region: "LON"
+    auth_plugin: "rackspace"
+    username: "joe.bloggs.rackspace"
+    api_key: "aaa-bbb-ccc-ddd"
+    auth_url: "https://identity.api.rackspacecloud.com/v2.0"
+version: 1
```

Comment:
Probably not the final location for these two sample files, right?  They look like they belong in the documentation somewhere.  Some minimal level of automated testing, maybe just a smoke test that some kind of object comes back from `AgentService.get_api` for these configurations (without raising an exception) would be good, too.